recipes/mbdata/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mbdata" %}
+{% set version = "26.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mbdata-{{ version }}.tar.gz
+  sha256: a0ada9081b29f72502aebb859589cfed39aac91eed7e6f944da9e90a3e5c547a
+
+build:
+  noarch: python
+  entry_points:
+    - mbslave=mbdata.replication:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python =>3.6

```

Comment:
Is `=>` OK or is this passing silently but in fact it is a bug? I always thought that only `>=` would work.

Reply:
CIs are passing. I guess that is OK. I don't think I ever tried to flip those :smile: